### PR TITLE
[DOC] add references in the docstring

### PR DESCRIPTION
**Pull request by gauzias**
Solves the issue #89

### CODE REVIEW

**Review by alexpron — COMMENTED** [review]
Thank you for you PR @gauzias good occasion to test the PR mechanism again.
1. Regarding the doc of spherical_mapping:
      + an extra line after the last reference will be clearer
      + for the type : may  be use specific type param field , its increase the volume of the documentation but make it clearer.

File: slam/mapping.py [hunk 1]
```diff
@@ -126,8 +85,13 @@ def spherical_mapping(mesh, mapping_type='laplacian_eigenvectors',
 def disk_conformal_mapping(mesh, lap_type='conformal',
                            boundary=None, boundary_coords=None):
     """
-    compute comformal mapping of a mesh to a disk
-    ADD ref
+    compute comformal mapping of a mesh to a disk, see the following references:
```
Comment (alexpron) [inline slam/mapping.py:88]: Computes

Comment (gauzias) [reply slam/mapping.py:88]: ok, well spotted!
Let me try to update the PR with only that change

Comment (alexpron) [reply slam/mapping.py]: Nice it worked apparently

**Pull request merged**